disnake/webhook/sync.py

```diff
@@ -652,9 +652,9 @@ def from_url(
 
         Returns
         -------
-        :class:`Webhook`
-            A partial :class:`Webhook`.
-            A partial webhook is just a webhook object with an ID and a token.
+        :class:`SyncWebhook`
+            A :class:`SyncWebhook` object with which it is possible
+            to handle webhooks synchronously.
```

Comment:
I'd say the old description here was mostly fine, except for the `Webhook`/`SyncWebhook` part; it's still a partial webhook after all